todo/Controllers/todoTasksController.cs

```diff
@@ -0,0 +1,96 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Todo.Models;
+using Todo.DTOs;
+using Todo.Services;
+using System.Security.Claims;
+
+namespace Todo.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class TodoTasksController : ControllerBase
+    {
+        private readonly ITodoTaskService _todoTaskService;
+
+        public TodoTasksController(ITodoTaskService todoTaskService)
+        {
+            _todoTaskService = todoTaskService;
+        }
+
+        [HttpGet]
+        public async Task<IActionResult> GetAllTodoTasks()
+        {
+            try
+            {
+                return Ok(await _todoTaskService.GetAllTodoTasksAsync());
+            }
+            catch
+            {
+                return StatusCode(500);
+            }
+        }
+        [HttpGet("id")]
+        public async Task<IActionResult> GetTaskById(int id)
+        {
+            var task = await _todoTaskService.GetTodoTasksAsync(id);
+            return task == null ? BadRequest() : Ok(task);
+        }
+        [HttpPost]
+
+        public async Task<IActionResult> PostTask(TodoTaskDTO input)
+        {
+            try
+            {
+                var newTaskId = await _todoTaskService.AddTodoTaskAsync(input);
+
+                return newTaskId == null ? BadRequest() : Ok(newTaskId);
+            }
+            catch
+            {
+                return BadRequest();
+            }
+        }
+        [HttpPut("{id}")]
+
+        public async Task<IActionResult> PutTask(int id, [FromBody] TodoTask input)
```

Comment:
```suggestion
        public async Task<IActionResult> PutTask(int id, [FromBody] TodoTask input)
```
sửa lại toàn bộ IActionResult thành ActionResult<{ResponseDTO}> nhé
dùng thế này swagger k gen ra response body được